packages/did/src/Did.chain.ts

```diff
@@ -404,13 +403,13 @@ export async function generateDidAuthenticatedTx({
  */
 export function didSignatureToChain(
   key: DidVerificationKey,
-  signature: Pick<DidSignature, 'signature'>
+  signature: Uint8Array
```

Comment:
This function didSignatureToChain is used in a single place, do we still need it?

Reply:
Not sure. Will users, who work with raw api calls need it?